### DISCUSSION TITLE
Adding Elestio as deployment option

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [joomla/joomla-cms#44064] [2. Cold Features]
Hey team,
I am Kaiwalya, Developer Advocate at Elestio. Elestio has been providing options of fully deploying and managing Joomla application as shown [here](https://elest.io/open-source/joomla). I think it would be a great idea if we can add it to official readme/documentation here.
🔔 In addition to this, if you are interested we provide partnership opportunities with tools we support by **revenue share** upon addition of this method in docs. If you would like to collaborate, just shoot me an email at [kaiwalya@elest.io](mailto:kaiwalya@elest.io) :)

Example Button:

[![Deploy on Elestio](https://elest.io/images/logos/deploy-to-elestio-btn.png)](https://elest.io/open-source/joomla)

**Comment by brianteeman** [comment]
Please see https://www.joomla.org/sponsor.html

**Comment by richard67** [comment]
Thanks for your proposal. An issue is not the right place for it. I’ll move it to discussions.
